src/pybind/mgr/run-tox.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
@tchaikov just removed all other `ron-tox.sh` files from the project, excep for `src/script/run_tox.sh`. Can you make use of the other one instead?